src/m365/search/commands/externalconnection/externalconnection-schema-add.spec.ts

```diff
@@ -0,0 +1,177 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Cli } from '../../../../cli/Cli';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { pid } from '../../../../utils/pid';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+const command: Command = require('./externalconnection-schema-add');
+
+describe(commands.EXTERNALCONNECTION_SCHEMA_ADD, () => {
+  const externalConnectionId = 'TestConnectionForCLI';
+  const schema = '{"baseType": "microsoft.graph.externalItem","properties": [{"name": "ticketTitle","type": "String"}]}';
+
+  let log: string[];
+  let logger: Logger;
+  let commandInfo: CommandInfo;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
```

Comment:
Hi, just popping in to notify you that we should update this line to:

```
sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
```

to align it with our latest change: #4215